### PR TITLE
Enable apt pinning

### PR DESCRIPTION
**Pull request by scaronni**
And make sure it works with people having the same setup as before.

See related mesos role PR: https://github.com/teralytics/ansible-mesos/pull/8

### CODE REVIEW

File: tasks/Debian.yml [hunk 1]
```diff
@@ -3,7 +3,13 @@
   apt_key: id=E56151BF keyserver=keyserver.ubuntu.com state=present
 
 - name: Add Mesosphere repo
-  apt_repository: repo="{{marathon_apt_repo}}" state=present update_cache=yes
+  apt_repository: repo='deb {{ marathon_apt_url }} {{ ansible_distribution_release | lower }} main' state=present update_cache=yes
```
Comment (lhoss) [inline tasks/Debian.yml]: use `marathon_os_distribution` instead of `ansible_distribution_release | lower`

File: defaults/main.yml [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 
+marathon_repo_host: repo.mesosphere.com
+
 # Debian: Mesosphere apt repository URL
-marathon_apt_package: "marathon={{ marathon_version }}-*"
-marathon_apt_repo: "deb http://repos.mesosphere.com/{{ansible_distribution|lower}} {{ansible_distribution_release|lower}} main"
+marathon_package_version: "1.0.540"
+marathon_os_distribution: "{{ ansible_distribution | lower }}"
+marathon_os_version: "{{ ansible_distribution_version.split('.') | join('') }}"
+marathon_apt_url: "http://{{ marathon_repo_host }}/{{ ansible_distribution | lower }}"
```
Comment (lhoss) [inline defaults/main.yml:10]: opt: could re-use `marathon_os_distribution`

File: defaults/main.yml [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
 marathon_version: "1.3.6"
 
+marathon_repo_host: repo.mesosphere.com
+
 # Debian: Mesosphere apt repository URL
-marathon_apt_package: "marathon={{ marathon_version }}-*"
-marathon_apt_repo: "deb http://repos.mesosphere.com/{{ansible_distribution|lower}} {{ansible_distribution_release|lower}} main"
+marathon_package_version: "1.0.540"
+marathon_os_distribution: "{{ ansible_distribution | lower }}"
+marathon_os_version: "{{ ansible_distribution_version.split('.') | join('') }}"
```
Comment (lhoss) [inline defaults/main.yml:9]: (opt) more standard to replace chars: `ansible_distribution_version | regex_replace('\.', '')`

**Review by lhoss — APPROVED** [review]

File: defaults/main.yml (hunk 3 above)
Comment (scaronni) [reply defaults/main.yml:9]: Same comment as above.

Comment (lhoss) [reply defaults/main.yml:9]: yeah you are right, this was taken from the (original) mesos role.. ok just keep as is 👍

**Review by chicofranchico — APPROVED** [review]

**Pull request merged**